app/src/main/java/org/stepik/android/presentation/course_content/CourseContentPresenter.kt

```diff
@@ -408,6 +414,34 @@ constructor(
             )
     }
 
+    fun getCalendarPrimaryItems() {
+        isBlockingLoading = true
+        compositeDisposable += courseCalendarInteractor
+                .getCalendarItems()
+                .subscribeOn(backgroundScheduler)
+                .observeOn(mainScheduler)
+                .doFinally {isBlockingLoading = false}
+                .subscribeBy(
+                    onSuccess = { view?.showCalendarChoiceDialog(it) },
+                    onError = { view?.showCalendarError(R.string.request_error)}
```

Comment:
If we always pass only `R.string.request_error` we can drop that argument.

Reply:
We pass a different string @ https://github.com/StepicOrg/stepik-android/pull/314/files#diff-67766caba8d2cf34b4a0b26a07108187R395

Reply:
Ok, can we switch to enum argument in this method in order to exclude string resources from presentation layer?